### DISCUSSION TITLE
Error analysis formulation in MTEX

### DISCUSSION THREAD

**Discussion by MohammadJoudivand** [mtex-toolbox/mtex#578] [Ask Anything]
Dear Guys
I am using MTEX to select a region from my experimental EBSD data as my RVE.

To justify the selection region I am using calcError function to quantify the magnitude of deviation of the encased and picked region from the main EBSD plot.

Are you able to kindly let me know how the error gets calculated when two ODFs are compared (the ODF of main plot and the ODF of selected region)?

Is there any descriptive file in terms of mathematics in this regard?

Waiting for your kind reply
Sincerely yours
Mohammad

**Comment by kilir** [comment]
Hi,
you can start with a look into `ODF/calcError`, starting at line 59.

Cheers,
Rüdiger

**Reply by MohammadJoudivand** [reply]
Are you able to be more specific?
Which line are you referring to?
Are you able to kindly provide a link for that?

**Reply by kilir** [reply]
The lines following from here, depending which error you compute:
 https://github.com/mtex-toolbox/mtex/blob/658741165aaf63efe691e53aafbb9d94654f43ea/ODFAnalysis/%40ODF/calcError.m#L59

**Reply by MohammadJoudivand** [reply]
Thank you Rüdiger
